incubator-tools/overlapping_split/overlapping_split.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<p>Fix the Markdown formatting, make into bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Acknowledge.</p>